pnet_datalink/Cargo.toml

```diff
@@ -10,7 +10,7 @@ keywords = ["networking", "datalink", "ethernet", "raw"]
 categories = ["network-programming"]
 
 [dependencies]
-libc = "0.2"
+libc = ">=0.2.34"
 ipnetwork = "0.12"
 pnet_base = { path = "../pnet_base", version = "0.20.0" }
 pnet_sys = { path = "../pnet_sys", version = "0.20.0" }
```

Comment:
Please bump the `pnet_sys` dependency to `0.20.1` so it depends on the version exporting the required constants.